GCEWindowsAgent/windowsaccounts.go

```diff
@@ -0,0 +1,288 @@
+//  Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Rename to windows_accounts.go

Reply:
and later, windows_accounts_windows.go :)